microsoft-edge/webview2/reference/javascript/hostobjectsasyncroot.yml

```diff
@@ -0,0 +1,61 @@
+### YamlMime:TSType
+name: HostObjectsAsyncRoot
+uid: 'WebView2Script!HostObjectsAsyncRoot:class'
+package: WebView2Script!
+fullName: HostObjectsAsyncRoot
+summary: >-
+  Contains asynchronous proxies for all host objects added via `CoreWebView2.AddHostObjectToScript` as well as options
+  to configure those proxies, and the container for synchronous proxies.
+
+
+  If you call `coreWebView2.AddHostObjectToScript("myObject", object);` in your host code then an asynchronous proxy for
+  `object` will be exposed in script as `chrome.webview.hostObjects.myObject`<!-- -->.
+remarks: ''
+isPreview: false
+isDeprecated: false
+type: class
+properties:
+  - name: options
+    uid: 'WebView2Script!HostObjectsAsyncRoot#options:member'
+    package: WebView2Script!
+    fullName: options
+    summary: Contains options applicable to `CoreWebView2.AddHostObjectToScript` added script proxies.
+    remarks: ''
+    isPreview: false
+    isDeprecated: false
+    syntax:
+      content: 'options: HostObjectsOptions;'
+      return:
+        type: '<xref uid="WebView2Script!HostObjectsOptions:class" />'
+  - name: sync
+    uid: 'WebView2Script!HostObjectsAsyncRoot#sync:member'
+    package: WebView2Script!
+    fullName: sync
+    summary: >-
+      Contains synchronous proxies for all host objects added via `CoreWebView2.AddHostObjectToScript`<!-- -->.
+
+
+      If you call `coreWebView2.AddHostObjectToScript("myObject", object);` in your host code then a synchronous proxy
+      for `object` will be exposed in script as `chrome.webview.hostObjects.sync.myObject`<!-- -->.
+    remarks: ''
+    isPreview: false
+    isDeprecated: false
+    syntax:
+      content: 'sync: HostObjectsSyncRoot;'
+      return:
+        type: '<xref uid="WebView2Script!HostObjectsSyncRoot:class" />'
+methods:
+  - name: cleanupSome()
+    uid: 'WebView2Script!HostObjectsAsyncRoot#cleanupSome:member(1)'
+    package: WebView2Script!
+    fullName: cleanupSome()
+    summary: Performs a best effort garbage collection on host object proxies that are no longer in use.
+    remarks: ''
+    isPreview: false
+    isDeprecated: false
+    syntax:
+      content: 'cleanupSome() : void;'
+      return:
+        type: void
+        description: ''

```

Comment:
fill in `description: ''`?

Reply:
The type is void and it returns nothing. It's okay to leave it blank.